tests/instancetype/instancetype.go

```diff
@@ -596,20 +595,17 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 			newVM := libvmi.NewVirtualMachine(newVMI,
 				libvmi.WithInstancetype(instancetype.Name),
 				libvmi.WithPreference(preference.Name),
+				libvmi.WithRunStrategy(virtv1.RunStrategyAlways),
```

Comment:
Again removes coverage of the `stopped` VM case we were previously asserting.

Reply:
Please elaborate

Reply:
As above the test would previously assert that `spec.{instancetype,preference}.revisionName` is populated before starting the VM.